src/main/java/com/google/devtools/build/lib/rules/cpp/CcCommon.java

```diff
@@ -354,54 +348,6 @@ public static ImmutableList<String> getCoverageFeatures(CppConfiguration cppConf
     return coverageFeatures.build();
   }
 
-  /**
-   * Creates a feature configuration for a given rule. Assumes strictly cc sources.
-   *
-   * @param ruleContext the context of the rule we want the feature configuration for.
-   * @param toolchain C++ toolchain provider.
-   * @return the feature configuration for the given {@code ruleContext}.
-   */
-  public static FeatureConfiguration configureFeaturesOrReportRuleError(
```

Comment:
Thanks for taking this on. The other 2 changes are dead indeed; this one, however, is still used by one internal test. Could you re-add it?

Reply:
I added the method back, inlined the dead one and added a `VisibleForTesting` annotation. PTAL.